metricbeat/module/jolokia/jmx/jmx.go

```diff
@@ -69,16 +62,19 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	http.SetMethod("POST")
 	http.SetBody(body)
 
+	log := logp.NewLogger(metricsetName)
```

Comment:
Given that there can be multiple instances of this metricset I recommend instantiating this `Logger` with context about the host. For example:

```
log := logp.NewLogger(metricsetName).With("host", base.HostData().Host)
```

Reply:
done